modules/tinymce/src/core/main/ts/init/Render.ts

```diff
@@ -45,9 +49,30 @@ const loadLanguage = (scriptLoader: ScriptLoader, editor: Editor) => {
   }
 };
 
-const loadTheme = (scriptLoader: ScriptLoader, editor: Editor, suffix: string): Promise<void> => {
+/**
+ * Initiate the download of both theme and model in parallel, then wait for both.
+ *
+ * Each has their own special case where nothing needs to be waited for.
+ */
+const loadThemeAndModel = (scriptLoader: ScriptLoader, editor: Editor, suffix: string): Promise<void> => {
```

Comment:
This logic probably needs reworking now sorry given the other change to load the theme in parallel. It could actually probably be a separate function now.